src/envoy/mixer/http_filter.cc

```diff
@@ -265,33 +265,30 @@ class Instance : public Http::StreamDecoderFilter,
 
   bool initiating_call_;
 
-  bool mixer_check_disabled_;
-  bool mixer_report_disabled_;
-
   // check mixer on/off flags in route opaque data
-  void check_mixer_route_flags() {
+  void check_mixer_route_flags(bool* check_disabled, bool* report_disabled) {
```

Comment:
is this a good style to pass bool pointers? Can this be refactored to "set_mixer_route_flags(flags)"?